src/main.rs

```diff
@@ -91,8 +91,7 @@ impl Blog {
                 ..ComrakOptions::default()
             };
 
-            let contents =
-                comrak::markdown_to_html(&contents[end_of_yaml + 5..], &options);
+            let contents = comrak::markdown_to_html(&contents[end_of_yaml + 5..], &options);
```

Comment:
I ran `cargo fmt` and it formatted a few lines outside of what I did.

Reply:
that's cool! i appreciate it